nomad/node_endpoint_test.go

```diff
@@ -2581,7 +2743,7 @@ func TestClientEndpoint_UpdateAlloc_NodeNotReady(t *testing.T) {
 	}
 	var allocUpdateResp structs.NodeAllocsResponse
 	err = msgpackrpc.CallWithCodec(codec, "Node.UpdateAlloc", allocUpdateReq, &allocUpdateResp)
-	require.ErrorContains(t, err, "not ready")
+	require.ErrorContains(t, err, "not allow to update allocs")
```

Comment:
I changed the error message in https://github.com/hashicorp/nomad/pull/15808/files#diff-86b976ad80dd125c60d191875f83e20d1e4ee971966b910665d75c3165cb65e4L1182-R1209